src/routes/files/handlers.js

```diff
@@ -0,0 +1,75 @@
+import { v4 as uuidv4 } from 'uuid';
+import handleErrors from '../../lib/apiErrorHandler';
+import { sequelize, File } from '../../models';
+import s3Uploader from '../../lib/s3Uploader';
+
+const fs = require('fs');
+const fileType = require('file-type');
+const multiparty = require('multiparty');
+
+const namespace = 'SERVICE:FILES';
+
+const logContext = {
+  namespace,
+};
+
+async function createFileMetaData(originalFileName, s3FileName, reportId) {
+  const newFile = {
+    activityReportId: reportId,
+    originalFileName,
+    key: s3FileName,
+    status: 'UPLOADING',
+  };
+  let file;
+  try {
+    await sequelize.transaction(async (transaction) => {
+      file = await File.create(newFile, transaction);
+    });
+    return file.dataValues;
+  } catch (error) {
+    return error;
+  }
+}
+const updateStatus = async (fileId, fileStatus) => {
+  let file;
+  try {
+    await sequelize.transaction(async (transaction) => {
+      file = await File.update({ status: fileStatus }, { where: { id: fileId } }, transaction);
+    });
+    return file.dataValues;
+  } catch (error) {
+    return error;
+  }
+};
+
+export default async function uploadHandler(req, res) {
+  const form = new multiparty.Form();
+  form.parse(req, async (error, fields, files) => {
+    if (error) {
+      res.status(500).send(error);
+    }
+    let buffer;
+    let metadata;
+    let fileName;
+    let type;
+
+    try {
+      const { path, originalFilename } = files.File[0];
+      const reportId = fields.reportID;
+      buffer = fs.readFileSync(path);
+      type = await fileType.fromFile(path);
```

Comment:
Interesting, I got an error here when running locally.
Used `curl localhost:8080/api/files -F File=@test.txt -F reportID=123456`

It looked like it was connecting ok and getting the path and originalFilename, but couldn't get the `type`.
It could be that I am on Windows. I am assuming this worked ok for you.

Reply:
Curl localhost:3000, not :8080. See if that works.

Reply:
I had an issue here as well. Looks like `file-type` [doesn't support txt files](https://www.npmjs.com/package/file-type#supported-file-types). I was able to upload a pdf. Maybe we should check for `undefined` here and send back an error.

Reply:
hmm, you're right. It also doesn't allow a csv. Will that be an issue? in #150, it says we need a list of acceptable file types.

Reply:
I imagine as long as we support images, pdfs and microsoft docs we should be okay. I see a lot of those extensions being supported, only issue might be older microsoft doc file formats are explicitly not supported.

Reply:
Do we have a list of file extensions that are supported/not-supported?

Reply:
I don't know if the goal here is to restrict what types of files can be uploaded, but if we want to allow people to upload plain text files (csvs, etc), we could remove the use of `fileType`, since that does magic file detection on binary files. 

I thought that browsers send the mime type of files when uploading, but I could be wrong.

Reply:
https://www.npmjs.com/package/file-type lists supported file types.

Reply:
I think this should be a larger discussion and maybe its own issue.